src/kitFilterHelper.ts

```diff
@@ -11,6 +11,10 @@ export default class KitFilterHelper {
         return generateHash(eventType + eventName);
     };
 
+    static hashEventMessage(eventName: string, eventType: valueof<typeof EventType>, messageType: valueof<typeof MessageType>): number {
```

Comment:
Technically the kitfilterhelpers are all related to side loaded kits. (This can probably be documented better in this file). You'll see them only ever used in the `sideloadedKit.ts` file.

I can see why you put hashEventMessage here, because there's a lot of other hashing going on here.  But I think it belongs in `utils.ts` instead.

Reply:
Docs about usage for the public api that ends up using the [KitFilterHelper](https://docs.mparticle.com/developers/client-sdks/web/kits/#filtering-data) as an FYI.